1809 uncomment 'delete process' test

constantin-ciobotaru opened the pull request:
https://github.com/Activiti/Activiti/issues/1809

Commit:
1809 uncomment 'delete process' test

https://github.com/Activiti/Activiti/issues/1809

salaboy reviewed (approved):
This is good of course, but I would like to add a test to see what happens when you try to execute operations such as Activate to a cancelled process? Can we make sure that we have a test in the acceptance tests that checks for such scenarios? We need to have an unified error handling mechanism for such state handling, can you please add the test and we can take it from there?

Commit:
1809 'try activate a cancelled process instance' scenario

- add scenario that tries to activate a cancelled process instance
- add ExpectedExceptionHandler and ExpectException annotation
- extend SerenityStories to allow performing ExpectingExceptionParametrisedStep
- add FeignErrorDecoder and ExpectRestError annotation
